### PR TITLE
Ts rewrite

### PR DESCRIPTION
**Pull request by x183**
Rewrite of lunch.chalmers.it in typescript, may have forgotten some functionality though

### CODE REVIEW

File: .env
Comment (Supergamer1337) [inline .env]: WHAT IS YOU .env DOING HERE?!?!?!?!?!?!?!?!?!?!?!?!?!?!?!?!??!?!?!?!

File: .gitignore
Comment (Supergamer1337) [inline .gitignore]: What is old/, why does this need to be here?

File: src/index.js
Comment (Supergamer1337) [inline src/index.js]: What is your compile JS doing here?

**Review by Supergamer1337 — CHANGES_REQUESTED** [review]
Well I won't approve this unless you fix the .gitignore among other things (that is probably it though).

**Review by Supergamer1337 — CHANGES_REQUESTED** [review]
It seems it prints to the console *everytime* someone makes a request, with what that response would be. It would be better to print simply when getting new data, or if getting new data failed.

**Review by Supergamer1337 — APPROVED** [review]
Finally I can approve of this

File: README.md
Comment (rhedinh) [inline README.md:1]: Add how-to for building project maybe?

File: package-lock.json
Comment (hampusrs) [inline package-lock.json]: Remove package-lock

File: README.md
Comment (hampusrs) [inline README.md]: Add how-to for building project

**Review by hampusrs — CHANGES_REQUESTED** [review]
This would be nice

**Comment by hampusrs** [comment]
Also please install git in the repo :')

File: README.md
Comment (x183) [reply README.md]: Is fixed

**Comment by hampusrs** [comment]
LGTM

**Review by hampusrs — APPROVED** [review]

**Pull request merged**
